recipes/selectio/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "selectio" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/selectio-{{ version }}.tar.gz
+  sha256: c822a161b3db8a86200a434a8fd56f8808a1cd270c71138e9a21ba2bf11a7472
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - scikit-learn >=1.0
+    - numpy >=1.21
+    - pandas >=1.3.5
+    - pyyaml >=6.0
+    - scipy >=1.7.3
+    - matplotlib-base >=3.5
+
+test:
+  imports:
+    - selectio
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sebhaan/selectio
+  summary: Multi-model Feature Importance Scoring and Auto Feature Selection
+  license: LGPL-3.0
```

Proper SPDX identifier:
```suggestion
  license: LGPL-3.0-only
```